python/ray/autoscaler/_private/util.py

```diff
@@ -724,6 +724,18 @@ def filter_placement_group_from_bundle(bundle: ResourceBundle):
     return demand_lines
 
 
+def get_constraints_report(lm_summary: LoadMetricsSummary):
+    constraint_lines = []
+    for bundle, count in lm_summary.request_demand:
+        line = f" {bundle}: {count}+ from request_resources()"
+        constraint_lines.append(line)
+    if len(constraint_lines) > 0:
+        constraints_report = "\n".join(constraint_lines)
+    else:
+        constraints_report = " (no cluster constraints)"
```

Comment:
request_resources() is a DeveloperAPI, most users don't know it and I doubt they know what `cluster constraints` is.

Reply:
It is indeed hard to link `request_resources` with `cluster constraints`. How about `(no request_resources() constraints)`?